### PR TITLE
WIP: new hoc for discussion

### PR DESCRIPTION
**Pull request by chrisnojima**
@buoyad

### CODE REVIEW

File: shared/util/container.js [hunk 1]
```diff
@@ -11,7 +13,10 @@ const createImmutableEqualSelector = createSelectorCreator(defaultMemoize, I.is)
 
 const NullComponent = () => null
 
-export {connect} from 'react-redux'
+// flowlint deprecated-type:off
+const storybookableConnect = (name: string, mapStateToProps: *, mapDispatchToProps: *, mergeProps: *) =>
```
Comment (buoyad) [inline shared/util/container.js]: :sheep: It's a pretty generic thing this does (also names the component in the virtual dom), maybe `namedConnect` instead?

**Comment by chrisnojima** [comment]
- [ ] fix typing

**Comment by chrisnojima** [comment]
- [ ] pull in readme from desktop branch

**Comment by chrisnojima** [comment]
the typing of this thing is a nightmare

**Pull request closed without merging**